resources/js/src/app/components/itemList/filter/ItemPriceSlider.js

```diff
@@ -0,0 +1,67 @@
+import url from '../../../mixins/url'
+
+Vue.component("item-price-slider", {
+    mixins: [url],
+
+    props: [
+        "template",
+        "facet"
+    ],
+
+    data() {
+        return {
+            priceFrom: "",
+            priceTo: ""
+        };
+    },
+
+    created() {
+        let self = this;
+
+        this.$options.template = this.template || "#vue-item-price-slider";
+
+        const values = this.getSelectedFilterValue(this.facet.id);
+
+        this.priceFrom = values ? values.min : this.facet.minPrice;
+        this.priceTo = values ? values.max : this.facet.maxPrice;
+
+        $(function() {
+            $("#price-slider-range").slider({
+                step: 1,
```

Comment:
I think our API provides a value for this. Consider getting it from the response XML and passing it here.